scripts/test_update_from_tag.sh

```diff
@@ -88,7 +88,7 @@ docker_pgcmd() {
 
 docker_pgscript() {
     local database=${3:-single}
-    docker_exec $1 "psql -h localhost -U postgres -d $database $PGOPTS -v ON_ERROR_STOP=1 -f $2"
+    docker_exec $1 "psql --set VERBOSITY=verbose --set ECHO=all -h localhost -U postgres -d $database $PGOPTS -v ON_ERROR_STOP=1 -f $2"
```

Comment:
Why do you need this change here? It does not seem warranted.

Reply:
It helped me debug this, so I thought I'd keep it. Is it better to remove it?

Reply:
Keep it. I was worried that it could cause flakes, but that is not the case.